database/migrations/2021_12_12_172840_create_orders_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrdersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->nullable(true)
+                ->references('id')->on('users')
+                ->cascadeOnUpdate()
+                ->nullOnDelete();
+            $table->foreignId('status_id')
+                ->references('id')->on('statuses')
+                ->cascadeOnUpdate()
+                ->restrictOnDelete();
+            $table->decimal('price',10,2)->nullable(false);
```

Comment:
by default in laravel everything is not nullable, so please ensure that you have desired result on all columns.

Reply:
Removed redundant nullable(false) from price column as well as from other migrations.